frontend/tsconfig.node.json

```diff
@@ -10,5 +10,11 @@
     "strict": true,
     "allowSyntheticDefaultImports": true
   },
-  "include": ["vite.config.ts", "eslint.config.ts", "vitest.config.ts"]
+  "include": [
+    "vite.config.ts",
+    "eslint.config.ts",
+    "vitest.config.ts",
+    "postcss.config.ts",
+    "tailwind.config.ts"
```

Comment:
Enabling typescript linting indicated that there was no tsconfig set-up for these two `.ts` files. 
I have added it to the `node` config, as they are not `app` related files (they are used only in development, not in the actual application)